charmcraft.yaml

```diff
@@ -0,0 +1,15 @@
+type: charm
+bases:
+    - build-on:
+        - name: ubuntu
+          channel: "20.04"
+          architectures: ["amd64"]
```

Comment:
Not sure if I've approved this before, but we should be consistent in the same yaml file whether we use list indents or list notation.

Reply:
agreed! thanks, this has been updated.